nabs/plans.py

```diff
@@ -476,8 +477,9 @@ def daq_ascan(detectors, motor, start, end, nsteps):
     end : int or float
         The last point in the scan.
 
-    nsteps : int
-        The number of points in the scan.
+    n : int or float
```

Comment:
This is clearly a better name than `nsteps` now that it isn't a number of steps, as much as I dislike single-variable names. I can't think of a better alternative.

Reply:
Is `step` (not `nsteps`) ambiguous enough that it could refer to either the position delta or the number of steps?

Just a thought, but I'm inclined to just say to leave it as-is.